doc/source/whatsnew/v0.16.0.txt

```diff
@@ -130,6 +130,33 @@ methods (:issue:`9088`).
     d    7
     dtype: int64
 
+ - default behavior for HDF write functions is now to keep rows that are all missing except for index. (:issue:`9382`)
+ 
+ Previously,
+
+  .. ipython:: python
+    In [1]: myFile = HDFStore('file.hdf')
```

Comment:
- show these examples with a ``DataFrame`` with a row that has all-nans, and one that doesnt.
- don't use camel case.
- use ``to_hdf/read_hdf`` (because for example you are not closing the store)